src/Adapter/Configuration/LogsConfiguration.php

```diff
@@ -52,9 +66,11 @@ class LogsConfiguration implements DataConfigurationInterface
      */
     private $validate;
 
-    public function __construct(ConfigurationInterface $configuration, TranslatorInterface $translator, Validate $validate)
+    public function __construct(Configuration $configuration, Context $shopContext, FeatureInterface $multistoreFeature, TranslatorInterface $translator, Validate $validate)
     {
         $this->configuration = $configuration;
+        $this->shopContext = $shopContext;
+        $this->multistoreFeature = $multistoreFeature;
```

Comment:
```suggestion
        parent::__construct($configuration, $shopContext, $multistoreFeature);
```

Reply:
You can remove attributes, too. They are all available in `abstract class AbstractMultistoreConfiguration`.